include/Foundation/NSFileManager.h

```diff
@@ -18,6 +18,7 @@
 #import <Foundation/FoundationExport.h>
 #import <Foundation/NSObject.h>
 #import <Foundation/NSPathUtilities.h>
+#import <Foundation/NSDictionary.h>
```

Comment:
@pradipd libclang is used to parse these headers in isolation. We don't have any apps that consume each header individually, so the problem gets hidden by the include order.

It is usually _important_ that each header stand alone, but it is only a _requirement_ because of libclang